tests/test_files_handling.py

```diff
@@ -16,13 +16,13 @@ def mock_files(request):
 
 
 def filter_only_jpeg(mock_files):
-    file_filter = ['jpg', 'jpeg']
-    return [file for file in mock_files if file[-3:] in file_filter ]
+    file_filter = ['jpg']
+    return [file for file in mock_files if file[-3:] in file_filter]
 
 
 @pytest.mark.parametrize("mock_files", [
     [],
-    ["test_file_long_name_with_numbers12345", "test_file_long_name_with_numbers12345.jpeg"],
```

Comment:
You have already included the .jpeg extension, please do not remove it from tests

Reply:
Done